### DISCUSSION TITLE
Failed to parse unknown error format

### DISCUSSION THREAD

**Discussion by SergejKko** [gabrie30/ghorg#258] [Q&A] [answered]
Hello, 

Thank you for the great tool!
I have successfully cloned all required repos 2 months ago. However, when I tried to run the same command again, I've got error:
**404 failed to parse unknown error format**
I've double-checked everything - access token is not expired, it do have enough permissions, base url is valid.
The command that I've been using is:
```
ghorg clone all-groups --base-url=https://comapny.gitlab.com/notboringproject/ --scm=gitlab --token=Os0mTok3n --preserve-dir --clone-type=org
```

Could you please give me a hint what can be wrong? 

Thank you in advance.

**Comment by gabrie30** [comment]
Hi @SergejKko could you try setting the base url to `--base-url=https://comapny.gitlab.com/notboringproject` and providing more of the logs from ghorg.

That error upon first glance doesn't seem to be coming from ghorg. What version of gitlab are you running and did anything change with the gilab environment recently?

[Here](https://github.com/profclems/glab/issues/81) is a similar error being thrown with gitlab due to the reverse proxy configs, I'm wondering if its similar to whats happening in your case.

**Comment by MaxG87** [comment]
I get the same error. I think it is due to the company using Active Directory to provide a Single-Sign-On login page to its hosted Gitlab.

I ran the following command. There is no `~/ghorg/` directory and thus no configuration file.

`ghorg clone all-groups --base-url="https://gitlab.ad.company.com/dev-group/" --scm=gitlab --token="super-secret-token" --preserve-dir`

The logs you asked for follow. I couldn't find an option to tune up verbosity, though.
```

 +-+-+-+-+ +-+-+ +-+-+-+-+-+
 |T|I|M|E| |T|O| |G|H|O|R|G|
 +-+-+-+-+ +-+-+ +-+-+-+-+-+

*************************************
* SCM           : gitlab
* Type          : org
* Protocol      : https
* Location      : /home/user/ghorg/
* Concurrency   : 25
* Base URL      : https://gitlab.ad.company.com/dev-group/
* Preserve Dir  : true
* Config Used   : none
* Ghorg version : v1.9.6
*************************************
Encountered an error, aborting
error getting groups error: GET https://gitlab.ad.company.com/users/sign_in: 404 failed to parse unknown error format:
```

**Comment by MaxG87** [comment, marked as answer]
I noticed that I specified the wrong URL. When using `ghorg clone all-groups` one must not give URL that points to a group itself. That wasn't to obvious. Anyhow, I could get `ghorg` working by using `--base-url="https://gitlab.ad.company.com/".
